src/media/js/main.js

```diff
@@ -61,7 +62,7 @@ define('main', [
     });
 
     z.page.on('navigate', function(e, url) {
-        if ([urls.reverse('login'), urls.reverse('fxa_authorize')].indexOf(url)) {
+        if ([urls.reverse('login'), urls.reverse('core/fxa_authorize')].indexOf(url)) {
```

Comment:
I just realised that this wasn't working after my "fix" and debugged it and then realised I read your comment wrong. Sigh.